### PR TITLE
refactor(autoware_launch): clean up component launch

### PR DESCRIPTION
**Pull request by takayuki5168**
## Description

clean up autoware.launch.xml regarding component launch call.

- planning: https://github.com/autowarefoundation/autoware_launch/pull/185
- control: https://github.com/autowarefoundation/autoware_launch/pull/186

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by takayuki5168** [comment]
planning simulator worked well

File: autoware_launch/launch/components/tier4_map_component.launch.xml [hunk 1]
```diff
@@ -1,11 +1,8 @@
 <?xml version="1.0"?>
 <launch>
-  <arg name="lanelet2_map_path"/>
-  <arg name="pointcloud_map_path"/>
-
   <include file="$(find-pkg-share tier4_map_launch)/launch/map.launch.py">
-    <arg name="lanelet2_map_path" value="$(var lanelet2_map_path)"/>
-    <arg name="pointcloud_map_path" value="$(var pointcloud_map_path)"/>
+    <arg name="lanelet2_map_path" value="$(var map_path)/$(var lanelet2_map_file)"/>
+    <arg name="pointcloud_map_path" value="$$(var map_path)/(var pointcloud_map_file)"/>
```
Comment (kminoda) [inline autoware_launch/launch/components/tier4_map_component.launch.xml:5]:
```suggestion
    <arg name="pointcloud_map_path" value="$(var map_path)/$(var pointcloud_map_file)"/>
```

**Review by kminoda — COMMENTED** [review]
Almost LGTM except the following. Thanks!

File: autoware_launch/launch/components/tier4_map_component.launch.xml (hunk 1 above)
Comment (takayuki5168) [reply autoware_launch/launch/components/tier4_map_component.launch.xml]: @kminoda Fixed. Thanks!

**Review by kminoda — APPROVED** [review]
LGTM

File: autoware_launch/launch/autoware.launch.xml [hunk 2]
```diff
@@ -63,50 +63,27 @@
...
-      <arg name="lanelet2_map_path" value="$(var map_path)/$(var lanelet2_map_file)"/>
-      <arg name="pointcloud_map_path" value="$(var map_path)/$(var pointcloud_map_file)"/>
-    </include>
+    <include file="$(find-pkg-share autoware_launch)/launch/components/tier4_map_component.launch.xml"/>
   </group>
 
   <!-- Sensing -->
   <group if="$(var launch_sensing)">
-    <include file="$(find-pkg-share tier4_sensing_launch)/launch/sensing.launch.xml">
-      <arg name="launch_driver" value="$(var launch_sensing_driver)"/>
```
Comment (miursh) [inline autoware_launch/launch/autoware.launch.xml:84]: @takayuki5168 I have one question. 
Why do you delete launch_driver argument?
In this case, how do you pass "launch_driver" to sensing.launch.xml?

Comment (takayuki5168) [reply autoware_launch/launch/autoware.launch.xml:84]: Sorry, I changed wrong.
I was planning to use https://github.com/autowarefoundation/autoware_launch/blob/main/autoware_launch/launch/components/tier4_sensing_component.launch.xml to call sensing.launch.xml.

I created a fix PR. Could you review and try it?
https://github.com/autowarefoundation/autoware_launch/pull/227

**Pull request merged**